client/sections.js

```diff
@@ -591,10 +591,11 @@ const sections = [
 	},
 	{
 		name: 'jetpack-cloud-manage-pricing',
-		paths: [ '/manage/pricing' ],
+		paths: [ '/manage/pricing', '/[^\\/]+/manage/pricing' ],
```

Comment:
Will the characters preceding `/manage/pricing` _only_ be the locale? If so, we can do something like https://github.com/Automattic/wp-calypso/blob/36995755c35c242fb29f630282072d96590e9081/client/sections.js#L120

Reply:
Funny you should ask! I considered a simpler variation of this yesterday [in a previous PR](https://github.com/Automattic/wp-calypso/pull/87201#discussion_r1478902099) but wasn't sure if it'd have full coverage. I then found `getLanguageRouteParam()` (p1707329362736969/1707328892.190799-slack-CTXBP902X), but wasn't able to use it in this file. I almost used the above regex, but figured it might be better to use the already-used convention...I prefer precision, but this is all new territory to me!

Reply:
I've switched it to use regex: 244a1f86fa